service/gov_pay.go

```diff
@@ -62,7 +62,7 @@ func (gp *GovPayService) GenerateNextURLGovPay(req *http.Request, paymentResourc
 	}
 
 	request.Header.Add("accept", "application/json")
-	request.Header.Add("authorization", "Bearer "+gp.PaymentService.Config.GovPayBearerToken)
+	request.Header.Add("authorization", "Bearer "+gp.PaymentService.Config.GovPayBearerTokenTreasury) //Hardcoded to Treasury
```

Comment:
Can we just add a TODO comment here saying that work to determine which token to use needs to be added?